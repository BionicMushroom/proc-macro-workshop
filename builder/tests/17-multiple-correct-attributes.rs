use derive_builder::Builder;

#[derive(Builder)]
struct Test {
    #[builder(each = "first")]
    #[builder(each = "second")]
    field: u32,
}

fn main() {}
