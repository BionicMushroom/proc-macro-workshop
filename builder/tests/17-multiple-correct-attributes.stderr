error: multiple `builder` attributes on the same field
 --> tests/17-multiple-correct-attributes.rs:6:5
  |
6 |     #[builder(each = "second")]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
